OpenRA.Test/OpenRA.Game/PriorityQueueTest.cs

```diff
@@ -53,7 +53,7 @@ sealed class PriorityQueueTest
 		public void PriorityQueueAddThenRemoveTest(int count, int seed)
 		{
 			var mt = new MersenneTwister(seed);
-			var values = Enumerable.Range(0, count);
+			var values = Enumerable.Range(0, count).ToList();
```

Comment:
The `Shuffle` method mutates the array it generates and doesn't change the callers input. It would be a behavioural change it we started to mutate the callers input in place, so we can't safely make that change.

An in-place array shuffle isn't a bad idea, but something that could be done separately.